deploy/ccm-networks.yaml

```diff
@@ -83,7 +83,7 @@ spec:
                 secretKeyRef:
                   name: hcloud
                   key: network
-          image: hetznercloud/hcloud-cloud-controller-manager:v1.13.2
+          image: hetznercloud/hcloud-cloud-controller-manager:v1.15.0 # x-release-please-version
```

Comment:
Did you manually add this? If so I think it will get overwritten on the next run of `script/barf-deploy-manifests-thingie.sh`, right?

Reply:
Good catch! Added it to the helm template so its included when we regenerate the templates.